dashboard/app/models/user.rb

```diff
@@ -568,6 +566,23 @@ def username_required?
     provider == User::PROVIDER_MANUAL
   end
 
+  def update_without_password(params, *options)
+    if params[:races]
+      update_columns(races: params[:races].join(','))
+      if params[:races].include? 'closed_dialog'
+        update_columns(races: 'closed_dialog')
+      end
+      if params[:races].length > 5
```

Comment:
This PR is decoupling the serialized attribute from the new columns, so as to allow removal of the former. As such, I'm currently keeping the sanitization in both places (`update_without_password` does the columns, `sanitize_and_set_race_data` does the serialized attribute. As part of deprecating the serialized attribute, the latter will be removed.